versions.js

```diff
@@ -9,10 +9,21 @@ function createVersionLink(version) {
   var span = document.createElement("span");
   span.setAttribute("class", "dropdown-text");
   link.appendChild(span);
-  span.innerHTML = version.name;
+  span.innerHTML = versionName(version);
   return tag;
 }
 
+function versionName(version) {
+  if (version.name) {
+    return version.name;
+  }
+  var name = "Viash " + version.version;
+  if (version.version == "latest") {
```

Comment:
Are you sure that this is correct? I believe it should check for the value of the `json.latest`, not being equal to `"latest"`.

Reply:
Good point, derp :)